charts/seed-bootstrap/prometheus-rules/recording-rules.rules.yaml

```diff
@@ -3,17 +3,32 @@ groups:
   rules:
   # Recording rules for pod usage
   - record: seed:container_cpu_usage_seconds_total:sum_by_pod
-    expr: sum(rate(container_cpu_usage_seconds_total[5m])) by (pod, namespace)
+    expr: sum(rate(container_cpu_usage_seconds_total[5m])) by (pod, container, namespace)
```

Comment:
All container label values are `POD`.

Reply:
I get the right container names in the shoot prometheus. Is this different in the seed or aggregate prometheus?

Reply:
Strange, I see the correct container label values now. Maybe I didn't let prometheus run long enough with the new changes.